dist/profile/files/bind/jenkins.io.zone

```diff
@@ -2,7 +2,7 @@
 
 ; SOA Record
 @  1D  IN  SOA    ns1.jenkins-ci.org. tyler.monkeypox.org. (
-                2015122201 ; serial, todays date + todays serial #
+                2017030801 ; serial, todays date + todays serial #
```

Comment:
@olblak The reason you needn't update the serial for the SOA record is because that record doesn't need to change for this. The SOA record is only (AFAIK) used to describe the authority of the zone, which has long since been established as `ns1.jenkins-ci.org`. and that's not changing here.

I don't think updating it is necessarily bad, but unnecessary when adding new unrelated A or CNAME records.